embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceBody.kt

```diff
@@ -0,0 +1,13 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Serializable representation of [EmbraceBody]
+ */
+@JsonClass(generateAdapter = true)
+internal data class EmbraceBody(
```

Comment:
The name of this class might be confused with body types in other objects. 
We can consider renaming to `EmbraceLogBody`, or you expect this to be reused in other types?

Reply:
I used this name because the OTel class is named only `Body`, but it's true that the name can be confusing. It is not used outside of the `opentelemetry.sdk.logs` package, I'll rename it.